org.civicrm.reqangular/src/common/services/session.js

```diff
@@ -42,4 +42,4 @@
       };
     }]);
   });
-})(CRM, Drupal);
+})(CRM, window.Drupal);
```

Comment:
Before you would get an error when including the module, just including it, not even using it.

Now you won't get an error if you include it, but yes, you will get an error if you try to use the module.

An error should be displayed if the logged in contact id is not defined, but this type of error will be confusing, so yes, let me refactor the bit inside.